op-service/testlog/capturing.go

```diff
@@ -55,16 +55,16 @@ func (r *CapturedRecord) Attrs(f func(slog.Attr) bool) {
 // CapturingHandler provides a log handler that captures all log records and optionally forwards them to a delegate.
 // Note that it is not thread safe.
 type CapturingHandler struct {
-	handler slog.Handler
-	Logs    *[]*CapturedRecord // shared among derived CapturingHandlers
+	slog.Handler
```

Comment:
Since all methods of `Handler` are implemented, I think it's fine to keep it a private field.

The methods on the wrapped handler are only accessed after finding the wrapped handler through the `Unwrap` method.

Reply:
Switching it back per your suggestion.